src/components/SupportRequestModal.js

```diff
@@ -19,22 +19,26 @@ const constraints = {
   description: { presence: { allowEmpty: false } }
 }
 
-const SupportRequestModal = ajaxCaller(class SupportRequestModal extends Component {
+const SupportRequestModal = compose(
+  ajaxCaller,
+  Utils.connectAtom(authStore, 'authState')
+)(class SupportRequestModal extends Component {
   constructor(props) {
     super(props)
+    const { contactEmail, email } = props.authState.profile
+
     this.state = {
       subject: '',
       description: '',
-      type: 'bug',
-      email: getUser().email
+      type: 'question',
```

Comment:
Is there a reason this is now "Question"? As I recall, @tmm211 and @knoblett wanted the default to be "Bug".

Reply:
Double checked with @bradtaylor on slack, we want this to default to question.